BoxContentSDK/BoxContentSDK/Requests/BOXRepresentationInfoRequest.m

```diff
@@ -0,0 +1,63 @@
+//
+//  BOXRepresentationInfoRequest.m
+//  BoxContentSDK
+//
+//  Created by Prithvi Jutur on 4/10/18.
+//  Copyright © 2018 Box. All rights reserved.
+//
+
+#import "BOXRepresentationInfoRequest.h"
+#import "BOXRequest_Private.h"
+#import "BOXRepresentation.h"
+#import "BOXAPIDataOperation.h"
+#import "BOXLog.h"
+#import "BOXDispatchHelper.h"
+#import "BOXRepresentationInfo.h"
+
+@interface BOXRepresentationInfoRequest ()
+@property (nonatomic, readwrite, strong) BOXRepresentation *representation;
+@end
+
+@implementation BOXRepresentationInfoRequest
+
+- (instancetype)initWithRepresentation:(BOXRepresentation *) representation {
+    self = [super init];
+    if (self != nil) {
+        _representation = representation;
+    }
+    return self;
+}
+
+- (BOXAPIOperation *)createOperation
+{
+    return [self JSONOperationWithURL:self.representation.infoURL
```

Comment:
Maybe evaluate self.representation.infoURL for nil

Reply:
What should the result for nil infoUrl be? Looking at precedents and it looks like we just do a BOXAssert when createOperation fails

Reply:
confirmed. Chatted offline. We do call the request and error. We can discuss with team to add incomplete operation error handling instead of making the request with an invalid url.